core/src/views.py

```
@@ -0,0 +1,355 @@
+        # Get guideline
+        guideline = get_object_or_404(Guideline, id=guideline_id, status='completed')
+
+        # Validate
+        try:
+            stored_guideline = guideline.to_stored_guideline()
+            validator = FastArticleValidator(stored_guideline, target_quartile=target_quartile)
+            result = validator.validate(content)
+
+            return Response(result.to_dict())
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)